processor/transformprocessor/README.md

```diff
@@ -51,6 +51,9 @@ transform:
   error_mode: ignore
   <trace|metric|log>_statements:
     - context: string
+      conditions: 
```

Comment:
Did you consider naming this configuration property `where` instead of `conditions`? The name `where` seems to nicely point out what it does in relation to the actual OTTL. Although `conditions` isn't bad. What do you think @TylerHelmuth @evan-bradley?

I suppose a good thing about `conditions` is that implies a list, whereas `where` does not. Also, when sorting alphabetically, `conditions` comes before `statements`, which helps make it harder to miss it. :smile:

Reply:
I was thinking about its relation to `filterottl` and the underlying struct using `Conditions` as the name, but I can see a case for `where` as well.

Reply:
>I suppose a good thing about `conditions` is that implies a list, whereas `where` does not.

I see arguments on both sides about the name, but in my opinion this point should be the tiebreaker. In the absence of any editor/IDE hints to the user about the type of this, I think we should lean toward using pluralization to indicate that this is a list. Unfortunately `wheres` doesn't sound great, so I think `conditions` is probably the best we can do.